src/SourceBuild/tarball/content/repos/arcade.proj

```diff
@@ -24,7 +24,6 @@
     <BuiltSdkPackageOverride Include="@(ArcadeSdkOverride)" />
     <BuiltSdkPackageOverride Include="@(ArcadeCoreFxTestingOverride)" />
     <BuiltSdkPackageOverride Include="@(ArcadePackagingOverride)" />
-    <BuiltSdkPackageOverride Include="@(ArcadeTargetFrameworkSdkOverride)" />
```

Comment:
Why was this removed?

Reply:
Arcade is no longer producing Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk. I have just realized it produces Microsoft.DotNet.Build.Tasks.TargetFramework instead. I am testing out changing the override locally now.